src/panel/NoResultPanel.jsx

```diff
@@ -13,7 +13,7 @@ const NoResultPanel = ({ resetInput }) => {
   };
 
   return (
-    <Panel close={close}>
+    <Panel close={close} fitContent="default">
```

Comment:
As declared in the `Panel` proptypes, `fitContent` should be an array. It works here because you're lucky `indexOf` exists on strings too, but it may break later with just a string.